src/ImageMagick.jl

```diff
@@ -98,11 +80,11 @@ function load_(file::Union{AbstractString,IO,Vector{UInt8}}; ImageType=Image, ex
         cs = "Gray"
     end
 
-    depth = getimagechanneldepth(wand, DefaultChannels)
+    depth = getimagedepth(wand)
     if depth <= 8
         T = UFixed8     # always use 8-bit for 8-bit and less
     else
-        T = ufixedtype[2*((depth+1)>>1)]  # always use an even # of bits (see issue 242#issuecomment-68845157)
+        T = UFixed{UInt16,((depth+1)>>1)<<1}  # always use an even # of bits (see issue 242#issuecomment-68845157)
```

Comment:
If ImageMagick ever supports images with more than 16 bits (does it now?), getting rid of the dictionary will make this harder to support. Any particular reason for hardcoding it here?